tools/expotools/src/commands/PublishPackages.ts

```diff
@@ -465,7 +464,7 @@ async function _preparePublishAsync(
 
   const packageJson = require(path.join(pkg.path, 'package.json'));
   const packageView = await _getPackageViewFromRegistryAsync(pkg.packageName);
-  const currentVersion = packageView && packageView.currentVersion;
+  const currentVersion = packageView?.currentVersion ?? null;
```

Comment:
I don't like this line :/ 
However, I had to add `?? null` cause otherwise `currentVersion` is `string | undefined`. 

Maybe we shouldn't use here the optional chaining?

Reply:
Let it be like you've proposed 👍